tests/system_metrics/test_collect_metrics.py

```diff
@@ -0,0 +1,82 @@
+from mlflow.system_metrics.metrics.cpu_monitor import CPUMonitor
+from mlflow.system_metrics.metrics.disk_monitor import DiskMonitor
+from mlflow.system_metrics.metrics.gpu_monitor import GPUMonitor
+from mlflow.system_metrics.metrics.network_monitor import NetworkMonitor
+
+
+def test_cpu_monitor():
+    cpu_monitor = CPUMonitor()
+    cpu_monitor.collect_metrics()
+
+    assert isinstance(cpu_monitor.metrics["cpu_utilization_percentage"], list)
+    assert isinstance(cpu_monitor.metrics["system_memory_usage_megabytes"], list)
+
+    cpu_monitor.collect_metrics()
+    aggregated_metrics = cpu_monitor.aggregate_metrics()
+    assert isinstance(aggregated_metrics["cpu_utilization_percentage"], float)
+    assert isinstance(aggregated_metrics["system_memory_usage_megabytes"], float)
+
+    cpu_monitor.clear_metrics()
+    assert len(cpu_monitor.metrics.keys()) == 0
```

Comment:
```suggestion
    cpu_monitor.clear_metrics()
    assert cpu_monitor.metrics == {}
```

I think this is simpler.

Reply:
good call